Libraries/LibWeb/HTML/ToggleEvent.h

```diff
@@ -31,13 +33,23 @@ class ToggleEvent : public DOM::Event {
     // https://html.spec.whatwg.org/multipage/interaction.html#dom-toggleevent-newstate
     String const& new_state() const { return m_new_state; }
 
+    // https://html.spec.whatwg.org/multipage/interaction.html#dom-toggleevent-source
+    GC::Ptr<DOM::Element> source() const
+    {
+        // The source getter steps are to return the result of retargeting source against this's currentTarget.
+        return as<DOM::Element>(retarget(m_source, current_target()));
```

Comment:
What happens if `m_source` is null? Doesn't `as` crash?

Reply:
`as<..>(nullptr)` returns `nullptr`